File: testresources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessStaticVariableIndirectIndirect_MethodVarVar.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessStaticVariableIndirectIndirect_MethodVarVar
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;

		public void Case8()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.GetServiceTwo().serviceOne.sName;
		}
	}
}

File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessStaticVariableIndirectIndirect_MethodVarVar.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessStaticVariableIndirectIndirect_MethodVarVar {

	private BackgroundService bckg;
	private String test;

	public AccessStaticVariableIndirectIndirect_MethodVarVar(){
		bckg = new BackgroundService();
	}
		//  Indirect-indirect access of a static attribute 
		@SuppressWarnings("static-access")
		public String case8() {
			test = bckg.getServiceTwo().serviceOne.sName;
			return test;
		}
}